### DISCUSSION TITLE
Multi-Tenant Sub & Custom Domain Support?

### DISCUSSION THREAD

**Discussion by ZainKhalidOfficial** [clerk/javascript#2269] [Support request - DO NOT OPEN HERE]
Does clerk auth supports such a multi-tenant web application where authentication is required for all sub and unique custom domains?
(app.mydomain.com, zain.mydomain.com, john.mydomain.com,
and
otherdomain.com, uniquedomain.ai, xyz.io, etc. )

If yes, then how? I can't find any documentation.

**Comment by jescalan** [comment]
If using the same Clerk app for all domains, it will work out of the box. If using a different Clerk app for each domain, we don't yet support this but will soon - [this is the project to track](https://feedback.clerk.com/roadmap/b4c407a7-4514-4ff6-85f7-39a8a645397b)

**Comment by Tags-collab** [comment]
#2269
